test/extensions/filters/http/aws_request_signing/aws_request_signing_filter_test.cc

```diff
@@ -72,6 +72,17 @@ TEST_F(AwsRequestSigningFilterTest, SignFails) {
   EXPECT_EQ(1UL, filter_config_->stats_.signing_failed_.value());
 }
 
+// Verify FilterConfigImpl's getters.
+TEST_F(AwsRequestSigningFilterTest, FilterConfigImplGetters) {
+  Stats::IsolatedStoreImpl stats;
+  auto signer = std::make_unique<Common::Aws::MockSigner>();
+  FilterConfigImpl config(std::move(signer), "prefix", stats, "foo");
+
+  config.signer();
+  config.stats();
```

Comment:
I think we should probably have some kind of expectation here. You could, for instance, `EXPECT_EQ(&stats, &config.stats())`. You could do something similar for signer if you grab the pointer (`signer.get()`) before moving it into the config. But if that seems too pedantic, I think `EXPECT_NO_THROW` would be reasonable.

Reply:
Sure.

Reply:
`EXPECT_EQ(&stats, &config.stats())` isn't what you think, since that returns the generated stats no the scope... so `NO_THROW` is probs good for that case.